assets/src/less/global-admin.less

```diff
@@ -255,12 +255,23 @@
 
     @media only screen and (max-width: 375px) {
         .dokan-admin-notices {
+            .notice-content{
+                flex-direction: column;
+                align-items: start !important;
+                gap: 1rem;
+            }
```

Comment:
This section is also used for *576px*. Could remove this replication?

Reply:
@media only screen and (max-width: 375px) for very small screen 
   @media screen and (min-width: 376px) and (max-width: 576px) for small screen.
   
   two separate scrreen bhai .
   
   as it is  not conflicted code ,we can skip.